modules/ossm-document-attributes.adoc

```diff
@@ -12,8 +12,8 @@
 :ProductName: Red Hat OpenShift Service Mesh
 :ProductShortName: Service Mesh
 :ProductRelease:
-:ProductVersion: 1.1.9
-:MaistraVersion: 1.1
+:ProductVersion: 2.0
```

Comment:
I think this should stay as 1.1.9 and 1.1 for this PR.

Reply:
Actually, I think this was supposed to change the name of this file to 2.x

Reply:
Good point, I just checked and the serverless docs reference our docs attributes. Updated.